### PR TITLE
Issue72- show-comment-single-star-Fatemeh 

### PR DESCRIPTION
**Pull request by FatemRahimi**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description
creating comments view in single star

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

**Comment by Mesgna616** [comment]
Hi Fatemeh, i just reviewed your code and it seems you haven't used the comments database we have created, the comments array should come with the stars as well . can you review your code in the meantime i will work on it as well

**Comment by FatemRahimi** [comment]
Thank you Jaspreet, I imported it, but I installed Sql again to remove
debug errors, I will do it again,

**Review by Mesgna616 — COMMENTED** [review]
Hi Fatemeh, on your  on your API end point i don't see a post method. How do you think the comment will be added to the database unless there is a way to post it ? can you please review it or please let me know how you approched it .

**Comment by FatemRahimi** [comment]
> Hi Fatemeh, on your on your API end point i don't see a post method. How do you think the comment will be added to the database unless there is a way to post it ? can you please review it or please let me know how you approched it .
Hey Mesgna, YesterdaY Postgres was deleted, I had no any data, Mark helped me to install again

File: client/src/pages/SingleStar.js [hunk 1]
```diff
@@ -9,101 +9,100 @@ import ListGroup from "react-bootstrap/ListGroup";
...
+                            {star.result}
+                        </Card.Text>
+                        <Card.Text>
+                            <span style={{ fontSize: "1rem", fontWeight: "bold" }}>
+                                Description:
+                            </span>{" "}
+                            {star.description}
+                        </Card.Text>
+                        <div>
+                        <Badge variant="secondary">{star.comment && star.comment.length}</Badge>{" "}
```
Comment (Mesgna616) [inline client/src/pages/SingleStar.js]: Hi Fatemeh, i don't think you need to implement the comment. length on the front end in order to access the number of comments left. We have already done it on the back end, thus once the stars are fetched from the back end, the number of comments will be included.  All we need to do is add a badge to it. You can refer to what I have done on a different branch

**Comment by Mesgna616** [comment]
Hi Fatemeh, i can see there are several branch conflicts. Can you go through it before we attempt to merge it .

File: client/src/pages/SingleStar.js (hunk 1 above)
Comment (FatemRahimi) [reply client/src/pages/SingleStar.js]: Hey Tsino, thank you for your review.I removed it.  I approved issue 72 today.

**Comment by FatemRahimi** [comment]
Thank you Tsino, how can I find this branch from my local machin.

**Pull request closed without merging**